regex-automata/src/nfa/thompson/pikevm.rs

```diff
@@ -1902,6 +1988,11 @@ impl Cache {
             stack: vec![],
             curr: ActiveStates::new(re),
             next: ActiveStates::new(re),
+            lookaround: {
+                let mut res = Vec::new();
+                res.resize(re.lookaround_count().as_usize(), None);
+                res
+            },
```

Comment:
Any particular reason for not using this?
```suggestion
            lookaround: vec![None; re.lookaround_count().as_usize()],
```

Reply:
Yes, it is called "ignorance", hahaha 😝